### PR TITLE
Add small updates after Fred review

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/marketplace.component.html [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
       </a>
       <a mat-list-item routerLink="wishlist" routerLinkActive="active">
         <mat-icon mat-list-icon  svgIcon="heart_fill"></mat-icon>
         <span mat-line>Wishlist</span>
       </a>
       <a mat-list-item routerLink="event" routerLinkActive="active">
-        <mat-icon mat-list-icon  svgIcon="video_library"></mat-icon>
-        <span mat-line>Sceenings</span>
+        <mat-icon mat-list-icon  svgIcon="screening"></mat-icon>
+        <span mat-line>My Sceenings</span>
```
Comment (Mussuranae) [inline apps/festival/festival/src/app/marketplace/marketplace.component.html:23]: Typo : scReenings

File: libs/utils/src/lib/static-model/staticModels.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ const models = {
     { 'slug': 'horror', 'label': 'Horror' },
     { 'slug': 'science-fiction', 'label': 'Science Fiction' },
     { 'slug': 'thriller', 'label': 'Thriller' },
-    { 'slug': 'coming-of-age', 'label': 'Coming of Age' },
+    { 'slug': 'coming-of-age', 'label': 'Young Adult' },
```
Comment (Mussuranae) [inline libs/utils/src/lib/static-model/staticModels.ts:11]: Can you also rename slug ?

Comment (GrandSchtroumpf) [reply libs/utils/src/lib/static-model/staticModels.ts:11]: It would change the database for a change of label. It's not worst it I think

Comment (pldespaigne) [reply libs/utils/src/lib/static-model/staticModels.ts:11]: good catch !

**Pull request merged**